docs/reference/errors-and-warnings/NU5128.md

```diff
@@ -18,7 +18,7 @@ f1_keywords:
 
 ### Issue
 
-The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the [Target Framework Moniker (TFM)](../target-frameworks.md) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers to believe the package is not compatible with the TFM.
+The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the [Target Framework Moniker (TFM)](../target-frameworks.md) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers to believe the package is not compatible with the TFM, if the package does not have dependencies. If the package has undeclared dependencies, the project using the package will experience runtime errors.
```

Comment:
```suggestion
The `lib/<tfm>/` or `ref/<tfm>/` directory in the package contains at least one file for the [Target Framework Moniker (TFM)](../target-frameworks.md) specified in the warning message. However, no dependency group exists for this TFM in the `nuspec` file. This may cause package consumers to believe the package is not compatible with the TFM, even if the package does not have dependencies. If the package has undeclared dependencies, the project using the package will experience runtime errors.
```